packages/office-ui-fabric-react/src/components/Button/Button.types.ts

```diff
@@ -73,6 +73,8 @@ export interface IButtonProps extends React.AllHTMLAttributes<HTMLAnchorElement
    */
   className?: string;
 
+  classNames?: {};
```

Comment:
this type is not quite right and needs comments.

Reply:
I'll change the typing but esentially I did this... to provide the classNames so they would merge properly and avoid the issue we discussed last friday. Do you think this is the right fix or there is a better way?